src/login/login.html

```diff
@@ -0,0 +1,23 @@
+<form>
+  
+    <div class="container">

```

Comment:
Maybe we can add a h1 tag that tells the user to login or something?

Reply:
As of right now, the login page is for testing purposes only, so I feel the tag would superfluous for this